cloud_build/verify-provenance.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+# Download the jq binary in order to obtain the artifact registry url from the
+# docker image provenance.
+echo "Installing jq using curl..."
+curl -Lo jq \
+  "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64" \
+  && chmod +x jq
+
+# Download slsa-verifier in order to validate the docker image provenance.
+echo "Installing slsa-verifier using go..."
+go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.1
```

Comment:
Is it possible to use a full hash rather than a version?

Reply:
Updated to hash!